components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/tool-editor/views/source.js

```diff
@@ -15,9 +15,9 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-define(['require', 'log', 'lodash', 'jquery', 'event_channel', 'app/source-editor/editor','ace/ace'
-    ],
-    function(require , log, _, $, EventChannel, SiddhiEditor,ace) {
+define(['require', 'log', 'lodash', 'jquery', 'event_channel', 'app/source-editor/editor', 'ace/ace',
+        'ace/ext/beautify/siddhi-rules'],
+    function (require, log, _, $, EventChannel, SiddhiEditor, ace, Beautify) {
```

Comment:
Shall we rename the variable with **camelCase**?

Reply:
Corrected in commit e30b1f4